### PR TITLE
Added Restaurant Api driven Logic and Cypress UI Tests

### PR DESCRIPTION
**Pull request by sirhazman07**

### CODE REVIEW

File: app/routes/api.js [hunk 1]
```diff
@@ -46,12 +46,13 @@ exports.bus = function(req, res){
 exports.restaurant = function(req, res){
    
     var routing = apiUrl + "restaurant";
-    console.log(routing);
+    console.log(routing);    
+    let questionPick = getRandomInt(3);
```
Comment (ale-sanchez-g) [inline app/routes/api.js:50]: API currently only has one question for restaurant, so before this works we need to make a change to the API.

**Comment by sirhazman07** [comment]
TEST SCREENSHOTS
API Testing with Cypress -- Validate the restaurant header (failed)
API Testing with Cypress -- Validate the restaurantQuestions body (failed)
COVID19 Battles -- Bus battle (failed)
COVID19 Battles -- Restaurant battle (failed)
COVID19 Mobile Battles -- Bus battle (failed)
COVID19 Mobile Battles -- Restaurant battle (failed)

File: app/routes/api.js (hunk 1 above)
Comment (sirhazman07) [reply app/routes/api.js:50]: all battlegrounds have been updated to 3 questions each

**Comment by ale-sanchez-g** [comment]
@sirhazman07 I notice the office test are failing

image

**Pull request merged**